renovate.json

```diff
@@ -4,8 +4,11 @@
     ":disablePeerDependencies",
     ":semanticCommitTypeAll(chore)",
     "group:allNonMajor",
-    "schedule:daily"
+    "schedule:weekly"
   ],
+  "major": {
```

Comment:
maybe?

Reply:
what's this do?

Reply:
i'm trying to get a master issue like https://github.com/gatsbyjs/gatsby/issues/16840

where you have to tick off individual major deps to upgrade, instead of having renovate create them for you automatically

on second thought, though, i don't think i want this. i'd like to eagerly do major bumps, at least here